src/stream/src/executor/dispatch.rs

```diff
@@ -607,6 +611,7 @@ impl HashDataDispatcher {
         output_indices: Vec<usize>,
         hash_mapping: ExpandedActorMapping,
         dispatcher_id: DispatcherId,
+        downstream_table_name: Option<String>,
```

Comment:
Obvisouly this is not *Hash* dispatcher. Please introduce a new dispatcher for your purpose.